src/components/Offering.vue

```diff
@@ -143,8 +141,10 @@ export default {
       currentOffering: null,
       message: "",
       model: 0,
+      a: 'require(',
```

Comment:
für was brauchen wir das?

Reply:
https://softwareengineering.stackexchange.com/questions/285288/is-the-use-of-one-letter-variables-encouraged